<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>95</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Up/Down</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLineEdit" name="lineEdit"/>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="maximumSize">
       <size>
        <width>95</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
    </item>
   </layout>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>215</y>
      <width>16</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>423</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogOut"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
  </widget>
  <action name="actionLogOut">
   <property name="text">
    <string>LogOut</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
